### PR TITLE
ZCS-911 Attach mail modal from compose view

### PR DESCRIPTION
**Pull request by tarangkhandelwal**
Changeset:
1. ZmMsg.Properties: Updating attachment size limit string to remove inline html for bold text.
2. zm.css:
   * Adding styles for attachment dialog.
   * Updated attach dialog width from 500px to 600px.
3. skin.properties:
   * Updated alternate selection background color to Appc (#fff).
   * Added styling section for dialogs.
4. Updated icons with svg versions.

### CODE REVIEW

File: WebRoot/img/zimbra/ImgLeftArrow.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="32" height="32" viewBox="0 0 32 32">
```
Comment (silentsakky) [inline WebRoot/img/zimbra/ImgLeftArrow.svg]: if possible remove height/width, as we are not using it and it becomes easier to identify icon files in file preview

File: WebRoot/messages/ZmMsg.properties [hunk 2]
```diff
@@ -329,8 +329,8 @@ attachment = Attachment
 attachmentConfirmRemove = Are you sure you want to permanently remove this attachment?
 attachmentConfirmRemoveAll = Are you sure you want to permanently remove all attachments?
 attachmentHeaderToolTip = Sort By Attachment
-attachmentLimitMsg = <b>Note:&nbsp;</b>Attachments may not be larger than {0}
-attachmentLimitMsgSingleFile = <b>Note:&nbsp;</b>Attachment size limit is {0}
+attachmentLimitMsg = Note:&nbsp;Attachments may not be larger than {0}
```
Comment (SatyamKumarOnline) [inline WebRoot/messages/ZmMsg.properties]: Spacing before 'Attachment' word.

File: WebRoot/skins/_base/base4/skin.properties [hunk 3]
```diff
@@ -1736,3 +1736,8 @@ CheckBoxTableCellSpacing    = 6px
 CheckboxTableLeftText       = text-align:right; margin-right:@CheckBoxTableCellSpacing@;
 CheckboxTableRightText      = text-align:left; margin-left:@CheckBoxTableCellSpacing@;
 
+###################
+#  Dialogs
```
Comment (SatyamKumarOnline) [inline WebRoot/skins/_base/base4/skin.properties]: Can we move dialog related below property to the same section in the property file.
Preferably, DialogBodyText , DialogBody

File: WebRoot/messages/ZmMsg.properties (hunk 2 above)
Comment (tarangkhandelwal) [inline WebRoot/messages/ZmMsg.properties]: just removed the <b> styling. Rest same as before.

File: WebRoot/skins/_base/base4/skin.properties (hunk 3 above)
Comment (tarangkhandelwal) [inline WebRoot/skins/_base/base4/skin.properties]: Sure . Moving it with the DialogBody property.

File: WebRoot/img/zimbra/ImgLeftArrow.svg (hunk 1 above)
Comment (tarangkhandelwal) [inline WebRoot/img/zimbra/ImgLeftArrow.svg]: done

**Pull request merged**